src/main/java/uk/gov/hmcts/reform/idam/api/fr/idm/manage/user/model/IdmUser.java

```diff
@@ -661,6 +661,12 @@ public void setConsentedMappings(List<List<Object>> consentedMappings) {
     this.consentedMappings = consentedMappings;
   }
 
```

Comment:
this is perfect there is just one more class you need to add this to - IdmESUser

the only difference in that class is that its indexable in elastic search